package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gulp-jade",
+  "name": "gulp-pug",
```

Comment:
he isn't giving it up, so switch this back to gulp-jade, gulp-pug will be blacklisted.